docs/releases/5.0.txt

```diff
@@ -291,6 +291,19 @@ Models
   :ref:`Choices classes <field-choices-enum-types>` directly instead of
   requiring expansion with the ``choices`` attribute.
 
+* :meth:`.Model.save` now passes the ``force_insert`` flag when saving models
+  on inherited parents. This ensures that all parent models are saved according
+  to the ``force_insert`` parameter.
+
+* The new ``Model._meta.get_non_abstract_parent_list()`` method is a new
+  addition to the model's meta options. It returns all the non-abstract parent
+  classes of the model as a list.
+
+* ``Model.queryset.create()`` method now sets the ``force_insert`` parameter to
+  ``False`` when calling the :meth:`.Model.save` method if the model is
+  inherited from a non abstract parent class. This allows the creation of
+  inherited models without violating constraints.
```

Comment:
These are not breaking changes but bugfixes and we don't document bugfixes in "feature" releases. Please revert changes to release notes.